test/expected/plan_expand_hypertable_optimized.out

```diff
@@ -421,10 +433,242 @@ SELECT * FROM cte ORDER BY value;
+(7 rows)
+
+--NOT IN - does not filter chunks
+:PREFIX SELECT * FROM hyper_w_space WHERE time < 10 AND device_id NOT IN ('dev5','dev6') ORDER BY value;
+                                      QUERY PLAN                                      
+--------------------------------------------------------------------------------------
+ Sort
+   Sort Key: hyper_w_space.value
+   ->  Append
+         ->  Seq Scan on hyper_w_space
+               Filter: (("time" < 10) AND (device_id <> ALL ('{dev5,dev6}'::text[])))
+         ->  Seq Scan on _hyper_2_106_chunk
+               Filter: (("time" < 10) AND (device_id <> ALL ('{dev5,dev6}'::text[])))
+         ->  Seq Scan on _hyper_2_104_chunk
+               Filter: (("time" < 10) AND (device_id <> ALL ('{dev5,dev6}'::text[])))
+         ->  Seq Scan on _hyper_2_105_chunk
+               Filter: (("time" < 10) AND (device_id <> ALL ('{dev5,dev6}'::text[])))
+         ->  Seq Scan on _hyper_2_103_chunk
+               Filter: (("time" < 10) AND (device_id <> ALL ('{dev5,dev6}'::text[])))
+(13 rows)
+
+--IN statement with subquery - does not filter chunks
+:PREFIX SELECT * FROM hyper_w_space WHERE time < 10 AND device_id IN (SELECT 'dev5'::text) ORDER BY value;
+                                  QUERY PLAN                                  
+------------------------------------------------------------------------------
+ Sort
+   Sort Key: hyper_w_space.value
+   ->  Nested Loop
+         ->  HashAggregate
+               Group Key: 'dev5'::text
+               ->  Result
+         ->  Append
+               ->  Seq Scan on hyper_w_space
+                     Filter: (("time" < 10) AND (('dev5'::text) = device_id))
+               ->  Seq Scan on _hyper_2_106_chunk
+                     Filter: (("time" < 10) AND (('dev5'::text) = device_id))
+               ->  Seq Scan on _hyper_2_104_chunk
+                     Filter: (("time" < 10) AND (('dev5'::text) = device_id))
+               ->  Seq Scan on _hyper_2_105_chunk
+                     Filter: (("time" < 10) AND (('dev5'::text) = device_id))
+               ->  Seq Scan on _hyper_2_103_chunk
+                     Filter: (("time" < 10) AND (('dev5'::text) = device_id))
+(17 rows)
+
+--ANY
+:PREFIX SELECT * FROM hyper_w_space WHERE time < 10 AND device_id = ANY(ARRAY['dev5','dev6']) ORDER BY value;
+                                     QUERY PLAN                                      
+-------------------------------------------------------------------------------------
+ Sort
+   Sort Key: hyper_w_space.value
+   ->  Append
+         ->  Seq Scan on hyper_w_space
+               Filter: (("time" < 10) AND (device_id = ANY ('{dev5,dev6}'::text[])))
+         ->  Seq Scan on _hyper_2_106_chunk
+               Filter: (("time" < 10) AND (device_id = ANY ('{dev5,dev6}'::text[])))
+         ->  Seq Scan on _hyper_2_105_chunk
+               Filter: (("time" < 10) AND (device_id = ANY ('{dev5,dev6}'::text[])))
+(9 rows)
+
+--ANY with intersection
```

Comment:
let's add a test with ANY with intersection, where that intersection should be empty.